### PR TITLE
Add timestamps to reaction events from IG during backfill

### PR DESCRIPTION
**Pull request by smweber**

### CODE REVIEW

File: mautrix_instagram/portal.py [hunk 1]
```diff
@@ -1265,7 +1267,10 @@ async def _handle_instagram_reactions(
                 continue
             puppet = await p.Puppet.get_by_pk(new_reaction.sender_id)
             intent = puppet.intent_for(self)
-            reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji)
+            if timestamp:
+                reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji, timestamp=timestamp)
+            else:
+                reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji)
```
Comment (sumnerevans) [inline mautrix_instagram/portal.py:1273]: does this work if you just do 

```suggestion
            reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji, timestamp=timestamp)
```

If it's `None`, does it just exclude the key?

**Review by tulir — APPROVED** [review]
(I changed most of it but it works now :D)

**Comment by smweber** [comment]
LOL thanks Tulir, I planned to test and fix it I promise ;)

File: mautrix_instagram/portal.py (hunk 1 above)
Comment (smweber) [reply mautrix_instagram/portal.py]: I guess it does ;)

**Pull request merged**